components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -885,15 +890,31 @@ public AccessTokenDO getAccessToken(String accessTokenIdentifier, boolean includ
 
             if (includeExpired) {
                 if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
-                    sql = SQLQueries.RETRIEVE_ACTIVE_EXPIRED_ACCESS_TOKEN_IDP_NAME;
+                    if (isTenantQualifiedUrlsEnabled && tenantDomain != null) {
```

Comment:
Shall we use StringUtils to check the null values in strings?

Reply:
Removed this from latest commits